frontends/common/resolveReferences/resolveReferences.cpp

```diff
@@ -404,10 +404,15 @@ void ResolveReferences::postorder(const IR::Type_Method *t) {
 
 bool ResolveReferences::preorder(const IR::Type_Extern *t) {
     refMap->usedName(t->name.name);
-    addToContext(t->typeParameters); return true; }
+    // FIXME -- should the typeParamters be part of the extern's scope?
```

Comment:
I don’t think it will work without them.

Reply:
Currently they're not part of the Type_Extern's scope, so need to be added as a separate scope (which is what this part of the change does)